ci/pipeline

```diff
@@ -201,12 +201,7 @@ def master(): Unit = {
   buildAndTestDockerAndLinuxPackages()
 
   // Uploads
-  val maybeArtifact = uploadTarballPackagesToS3(version, "builds")
-  maybeArtifact.foreach { artifact =>
```

Comment:
this is something that was not removed on the master branch here https://github.com/mesosphere/marathon/pull/5775/files

was it intentional to remove this?